IdentityServer/v7/docs/content/fundamentals/license_key.md

```diff
@@ -1,42 +1,115 @@
 ---
-title: "License Key"
+title: "Licensing"
 weight: 60
 ---
 
-When deploying your IdentityServer to production, you will need to configure your license key.
-This can be configured in one of two ways:
+IdentityServer is free for development, testing and personal projects, but production use
+requires a [license](https://duendesoftware.com/products/identityserver). 
+
+## Editions
+There are three license editions which include different [features](https://duendesoftware.com/products/features).
+
+#### Starter Edition
+The Starter edition includes the core OIDC and OAuth protocol implementation. This is an
+economical option that is a good fit for organizations with basic needs. It's also a great
+choice if you have an aging IdentityServer4 implementation that needs to be updated and
+licensed. The Starter edition includes all the features that were part of
+IdentityServer4, along with support for the latest .NET release, improved observability
+through OTEL support, and years of bug fixes and enhancements. 
+
+#### Business Edition
+The Business edition adds additional features that go beyond the core protocol support
+included in the Starter edition. This is a popular license because it adds the most
+commonly needed tools and features outside a basic protocol implementation. Feature
+highlights include our backend-for-frontend security framework for SPAs, support for
+server side sessions, and automatic signing key management. 
+
+#### Enterprise Edition
+Finally, the Enterprise edition includes everything in the Business edition and adds
+support for features that are typically used by enterprises with particularly complex
+architectures or that handle particularly sensitive data. Highlights include resource
+isolation, the OpenId Connect CIBA flow, and dynamic federation. This is the best option
+when you have a specific threat model or architectural need for these features.
+
+## License Key
+The license key can be configured in one of two ways:
 * Via a well-known file on the file system
 * Programmatically in your startup code
 
-## File System
+#### File System
 
-Duende IdentityServer will look for a file called *Duende_License.key* in the same directory as your hosting application.
-If present, the contents of the file will be loaded as the license key.
+IdentityServer looks for a file named either *Duende_License.key* or
+*Duende_IdentityServer_License.key* in the
```

Comment:
let's omit the older file name from here. we wanted to move away from the one we used to use.